conftest.py

```diff
@@ -204,6 +236,26 @@ def test_run_benchmark(benchmark_db_session, request, testrun_uid):
 
 @pytest.fixture(scope="function")
 def benchmark_time(test_run_benchmark):
+    """Benchmark the wall clock time of executing some code.
```

Comment:
```suggestion
    """Benchmarks the wall clock time of executing some code.
```

Reply:
Good to know, I was not aware of this. Thanks for the clarifications.

Reply:
FWIW, I was just made aware by @ntabris that the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) recommends the opposite:
> The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable.""")

So both styles are being used in different places. Since https://numpydoc.readthedocs.io/en/latest/format.html refers to PEP 257 but not Google, I'd give that one precedence.